krankerl.toml

```diff
@@ -0,0 +1,5 @@
+[package]
+before_cmds = [
+	"npm install --deps",
```

Comment:
maybe this should be changed to `npm ci` to keep consistency between all integrations  that uses krankel.toml?